### ISSUE TITLE
Update curation policy

### ISSUE THREAD

**Issue by lwaldron** [waldronlab/BugSigDB#90]
Some documentation items to add to https://bugsigdb.org/Curation_Policy @ftzohra22. @tosfos some of these might be of interest to you just for curiosity about NCBI naming and synonyms, and how specific names can change over time. It's good we are storing taxids and looking up names, so I assume it shouldn't be an issue e.g. when a species is moved to another genus. 

1. If two different figures/tables differ _only_ in which taxonomic level they provide (e.g. one figure for species, one supplemental table for genera, but these resulted from the same contrast), these can be combined into a single signature.
2. If taxa are not already found in bugsigdb.org, autocomplete won't show them. Any time autocomplete doesn’t work, you have to search https://www.ncbi.nlm.nih.gov/taxonomy and enter the integer ID if you can find it there. I found it helpful to start typing the genus and species, then cut and paste it into the NCBI taxonomy browser if autocomplete didn’t show it. The NCBI taxonomy is too large for autocomplete, so bugsigdb.org only autocompletes things already in the wiki and instead expects either integer NCBI taxids, or uninterpreted text.
3. Where you see “Unknown” and a genus level, referring to an unknown species from a known genus, this is often referred to by adding “sp.” to the genus name. For example instead of “Unknown Subdoligranulum” I found “[Subdoligranulum sp.](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=2053618)” in the taxonomy browser (ID 2053618) and entered this ID into the bugsigdb.org signature
4. For species that are only classified to higher levels of the taxonomy than genus (for example only their family or order are known), try adding “bacterium” to the family in your NCBI taxonomy browser search, order etc. E.g.  see [Clostridiales bacterium](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=1898207&lvl=3&lin=f&keep=1&srchmode=1&unlock) (Taxonomy ID: 1898207, unclassified species from the Eubacteriales order). [Eubacteriales](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=186802&lvl=3&lin=f&keep=1&srchmode=1&unlock) is a heterotypic synonym for Clostridiales.
5. Sometimes the genus names were abbreviated in the signature, e.g. “F. nucleatum” instead of “Fusobacterium nucleatum”. This is supposed to be done only after the first mention of the genus/species binary name, but in this signature I couldn’t find the full name elsewhere, so the easiest way was to go to the NCBI taxonomy and look up the species.
6. The NCBI taxonomy names sometimes include square brackets, even if they are not part of the published signature. For example [[Ruminococcus] torques](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=33039&lvl=3&lin=f&keep=1&srchmode=1&unlock). At the bottom of this NCBI page it clarifies that “Square brackets ([ ]) around a genus indicates that the name awaits appropriate action by the research community to be transferred to another genus.” This is fine, the NCBI Taxonomy ID will remain and the genus will be automatically corrected in the future just by using the NCBI taxonomy. Note that bugsigdb.org stores NCBI Taxonomy IDs, even though it instead shows current names to the user.
7. Sometimes searching for a name in the NCBI taxonomy will take you instead to a synonym, which is fine. For example, try searching for “Bacteroides symbiosus”; it takes you to to the page for [[Clostridium] symbiosum](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?mode=Info&id=1512&lvl=3&lin=f&keep=1&srchmode=1&unlock), which lists Bacteroides symbiosus as a “[homotypic synonym](https://en.wiktionary.org/wiki/homotypic#:~:text=In%20botanical%20nomenclature%20a%20homotypic,synonym%20of%20the%20new%20name.)”. The important thing here is that you have the correct Taxonomy ID, which refers both to the current correct name and this homotypic synonym. There are other kinds of synonyms too, Google provides good definitions (usually the first one is from wiktionary).
8. Genus and higher-level taxonomic names (everything but species) are always capitalized. It can be convenient to start entering these names in bugsigdb.org uncapitalized, so that you know if it is successfully autocompleted by the introduction of capitalization. This is important because it’s possible to enter a valid name without autocomplete, that is never matched to the NCBI taxonomy in bugsigdb.org, and is instead treated as plain-text and not a part of the taxonomy.

**Comment by ftzohra22** [comment]
Policy updated.
